hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1018,11 +1017,37 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	// Set SupportsMultiArchNodePools capable condition
+	{
+		condition := meta.FindStatusCondition(hcluster.Status.Conditions, hyperv1.SupportsMultiArchNodePools)
+
+		if condition == nil || condition.ObservedGeneration != hcluster.Generation || condition.Status != metav1.ConditionTrue {
```

Comment:
why the check for condition.Status != metav1.ConditionTrue?

Reply:
That was a copy/paste leftover when I copied this from https://github.com/openshift/hypershift/blob/04e367810b6553909ea00e96c4ccb91e48f5dc5e/hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go#L1012.

Reply:
It's been removed